backend/src/Altinn.Authorizationadmin/Altinn.Authorizationadmin/Services/Implementation/PolicyAdministrationPoint.cs

```diff
@@ -518,11 +546,6 @@ private async Task<List<Rule>> DeleteRulesInPolicy(RequestToDelete rulesToDelete
 
             DelegationHelper.TryGetResourceFromAttributeMatch(rulesToDelete.PolicyMatch.Resource, out ResourceAttributeMatchType resourceMatchType, out string resourceRegistryId, out string org, out string app);
 
-            if (resourceMatchType != ResourceAttributeMatchType.AltinnApp)
```

Comment:
Same here as in DeleteAllRulesInPolicy:
Still should check the result of the TryGetResourceFromAttributeMatch operation. If it fails and/or the resourceMatchType is none operation should fail as the resource cannot be identified